TriblerGUI/tribler_window.py

```diff
@@ -140,11 +140,12 @@ def __init__(self):
         # If QWebEngineView cannot be imported, make sure to render the graph trust page instead.
         try:
             from PyQt5.QtWebEngineWidgets import QWebEngineView
-            self.trust_page_network.initialize_trust_page()
+            self.network_explorer_page.initialize_web_page()
         except ImportError:
-            self.trust_page_graph.initialize_trust_page()
+            self.trust_page_graph.initialize_web_page()
```

Comment:
Why is this changed? This will introduce a bug when the trust page graph is loaded.